ros2_generate_interface_docs/ros2_generate_interface_docs/main.py

```diff
@@ -120,6 +120,11 @@ def main(argv=sys.argv[1:]):
     parser.add_argument(
         '--outputdir', type=str, default='api',
         help='Output directory')
+    parser.add_argument(
+        '--packages-select',
+        default='',
```

Comment:
Using an empty string for an argument which contains a list is weird. For arguments containing multiple values either don't define a default (which will result in `None` if not specified) or use an empty list `[]`.